server/master/src/main/java/org/apache/accumulo/master/Master.java

```diff
@@ -178,6 +178,7 @@
 
   final static int ONE_SECOND = 1000;
   final static long TIME_TO_WAIT_BETWEEN_SCANS = 60 * ONE_SECOND;
+  final static long TIME_TO_CACHE_RECOVERY_WAL_EXISTENCE = 20 * ONE_SECOND;
```

Comment:
I was just making it consistent in style with the existing code that surrounds it.  Since this is 1.9 I will make as few changes as possible and focus on the bug fix.

Reply:
Could possibly fix it up in 2.1, as long as fixing the surrounding code also does not creep too much.

Reply:
Didn't realize this was 1.9. Most of my comments (such as those pertaining to lambdas) are based on that misunderstanding. Please disregard anything that doesn't apply.